CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+# 1.0.0-beta.5 - Februrary 22, 2021
+
+- Toggle widget and plugin 
```

Comment:
this is done for internal sample so not actually related to customer, should either remove, or mention it's only for the internal sample